### PR TITLE
Add Classes support for javascript Books

### PR DESCRIPTION
**Pull request by BucurEva87**
Add support for object oriented programming adding and removing of books.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,56 +1,87 @@
-let books = [];
-const title = document.querySelector('.title');
-const author = document.querySelector('.author');
-const displayContainer = document.querySelector('.container');
+// const { isClosingBraceToken } = require('eslint-utils');
```
Comment (codecaiine) [inline index.js]: - [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile:

File: index.js [hunk 2]
```diff
@@ -1,56 +1,87 @@
...
-    });
+class Book {
+  constructor(title, author) {
+    this.author = author;
+    this.title = title;
   }
-};
-display();
\ No newline at end of file
+}
```
Comment (codecaiine) [inline index.js]: - [ ] Good job so far for creating the class :+1:  but there are some issues you need to address :cry: From the requirement, your class has `methods` to `add `and `remove` books and you should ensure that these two methods inside your class are working fine :+1: 

See an example below :point_down: 

```javascript
class Books {
  constructor() {
    this.books = [];
  }

 // Add book method
  addBook(title, author) {
  }

 // Remove book method
 RemoveBook(bookID) {
  }

 // Get book method
  getBooks() {
    return this.books;
  }
}

```

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @BucurEva87 @Loltolo-Lesapiti , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Descriptive Pull Request :heavy_check_mark: 
- Good use of gitflow :heavy_check_mark: 
- Good commits history :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥✅✅ 

## Highlights ✨
- No Linters error ✅✅
- Good commit messages ✅✅
- Great working on the functionality ✅✅
- Descriptive pull request ✅✅
-  Professional README ✅✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**